addons/payment_stripe/data/payment_acquirer_data.xml

```diff
@@ -4,6 +4,8 @@
     <record id="payment.payment_acquirer_stripe" model="payment.acquirer">
         <field name="provider">stripe</field>
         <field name="allow_tokenization">True</field>
+        <field name="enable_express_checkout">True</field>
```

Comment:
@pierre-pde says that it's better to have it enabled by default since it will work for Google Pay. Only Apple Pay is affected by the extra configuration and at worst, only Apple Pay won't work and users will just have to follow the documentation.